akka-cluster/src/main/scala/akka/cluster/Gossip.scala

```diff
@@ -333,13 +333,6 @@ private[cluster] class GossipEnvelope private (
       ser = null
     }
   }
-
-  @throws(classOf[java.io.ObjectStreamException])
-  private def writeReplace(): AnyRef = {
-    deserialize()
-    this
-  }
-
 }
```

Comment:
This was for Java serialization. We don't support that for cluster messages so ok to remove.